speed-transition.lua

```diff
@@ -1,12 +1,32 @@
-lookahead = 5
-speedup = 2.5
-leadin = 1 --range 0-2
-skipmode = false
-directskip = false
----------------
+lookahead = 5         --if the next subtitle appears after this threshold then speedup
+speedup = 2            --the value that "speed" is set to during speedup
+leadin = 1            --seconds to stop short of the next subtitle
+skipmode = false      --instead of speeding up playback seek to the next known subtitle
+maxskip = 5            --max seek distance (seconds) when skipmode is enabled
+skipdelay = 1          --in skip mode, this setting delays each skip by x seconds (must be >=0)
+directskip = false      --seek to next known subtitle (must be in cache) no matter how far away
+dropOnAVdesync = true
+--Because mpv syncs subtitles to audio it is possible that if audio processing lags behind--
+--video processing then normal playback may not resume in sync with the video. If "avsync" > leadin--
+--then this disables the audio so that we can ensure normal playback resumes on time.
+ignorePattern = true  --if true, subtitles are matched against "subPattern". A successful match will be treated as if there was no subtitle
+subPattern = "^[#♯♩♪♬♫🎵🎶%[%(]+.*[#♯♩♪♬♫🎵🎶%]%)]+$"
+---------------User options above this line--
 
+readahead_secs = mp.get_property_native("demuxer-readahead-secs")
```

Comment:
storing the setting because we change it on Enable. Since the mpv cache rewrite from way back, the default is only 1 second which effectively prevents lookaheads on embedded subtitles.